product_docs/docs/epas/15/installing/windows/installing_advanced_server_with_the_interactive_installer/invoking_the_graphical_installer_from_the_command_line/performing_an_installation_with_limited_privileges.mdx

```diff
@@ -9,62 +9,54 @@ redirects:
 
 To perform an abbreviated installation of EDB Postgres Advanced Server without access to administrative privileges, invoke the installer from the command line and include the `--extract-only` option. The `--extract-only` option extracts the binary files in an unaltered form, allowing you to experiment with a minimal installation of EDB Postgres Advanced Server.
 
-If you invoke the installer with the `--extract-only` options, you can either manually create a cluster and start the service, or run the installation script. To manually create the cluster, you must:
+If you invoke the installer with the `--extract-only` options, you can either manually create a cluster and start the service or run the installation script. To manually create the cluster, you must:
 
--   Use `initdb` to initialize the cluster
--   Configure the cluster
--   Use `pg_ctl` to start the service
+-   Use `initdb` to initialize the cluster.
+-   Configure the cluster.
+-   Use `pg_ctl` to start the service.
 
-For more information about the `initdb` and `pg_ctl` commands, see the PostgreSQL Core Documentation for [initdb](https://www.postgresql.org/docs/current/static/app-initdb.html) and [pg_ctl](https://www.postgresql.org/docs/current/static/app-pg-ctl.html).
+For more information about the `initdb` and `pg_ctl` commands, see the PostgreSQL core documentation for [initdb](https://www.postgresql.org/docs/current/static/app-initdb.html) and [pg_ctl](https://www.postgresql.org/docs/current/static/app-pg-ctl.html).
 
-If you include the `--extract-only` option, the installer steps through a shortened form of the `Setup` wizard. During the brief installation process, the installer generates an installation script that can be later used to complete a more complete installation. You must have administrative privileges to invoke the installation script.
+If you include the `--extract-only` option, the installer steps through a shortened form of the setup wizard. During the brief installation process, the installer generates an installation script that you can use later for a more complete installation. You must have administrative privileges to invoke the installation script.
 
 The installation script:
 
 -   Initializes the database cluster if the cluster is empty.
```

Comment:
@nidhibhammar There are several places in here that say 14 that probably should say 15?